.pre-commit-config.yaml

```diff
@@ -1,77 +1,52 @@
+fail_fast: false
+default_language_version:
+  python: python3
+default_stages:
+  - commit
+  - push
+minimum_pre_commit_version: 2.16.0
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
-    hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
-        exclude: |
-          (?x)(
-              .github/workflows/latest-changes.jinja2
```

Comment:
Are you sure we don't need to exclude it?

Reply:
I think you need to add it back so that things don't break, I messed with this early on